.github/workflows/release-app.yml

```diff
@@ -26,7 +29,7 @@ jobs:
           api-key: ${{ secrets.CN_API_KEY }}
 
   publish-tauri:
-    needs: draft
+    needs: [draft, generate_changelog]
```

Comment:
I'm not sure this works as intended. I'm not 100% sure this will make the CHANGELOG available at this point. I think we'd need to actually run this to know, but I couldn't make it with act due to the complexity of this run

PS: The `generate-changelog.yml` itself was tested with act and worked as intended.